manager-rs/dauth-service/src/database/user_infos.rs

```diff
@@ -8,10 +8,12 @@ use crate::data::error::DauthError;
 pub async fn init_table(pool: &SqlitePool) -> Result<(), DauthError> {
     sqlx::query(
         "CREATE TABLE IF NOT EXISTS user_info_table (
-            user_info_id TEST PRIMARY KEY,
+            user_info_id TEXT NOT NULL,
```

Comment:
This is one of the first tables I created, and I clearly chose a bad naming scheme. I would have changed it, but I wanted to minimize the number of files touched by this PR. Since this is the only active branch though, I can changes this.